apps/frontend/src/app/5_pages/BitocracyPage/components/NewProposalForm/components/ProposalDataForm/ProposalDataForm.utils.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+export const generateFormGroupLabel = (
+  labelText: string,
+  value: string,
+  maxLength: number | string,
+) => {
+  return (
+    <div className="flex items-center justify-between w-full">
+      <div>{labelText}</div>
+      <div className="text-gray-30">
+        {value.length}/{maxLength}
+      </div>
+    </div>
+  );
+};
+
+export const formatProposalText = (
+  title: string,
+  discussionUrl: string,
+  summary: string,
+  proposalText: string,
```

Comment:
Can just accept entire form

```suggestion
{ title, link, summary, text }: ProposalDetails
```